Sofa/Component/Constraint/Lagrangian/Model/src/sofa/component/constraint/lagrangian/model/BilateralLagrangianConstraint.inl

```diff
@@ -46,9 +46,27 @@ BilateralLagrangianConstraint<DataTypes>::BilateralLagrangianConstraint(Mechanic
     , d_restVector(initData(&d_restVector, "rest_vector","Relative position to maintain between attached points (optional)"))
     , d_activate( initData(&d_activate, true, "activate", "control constraint activation (true by default)"))
     , d_keepOrientDiff(initData(&d_keepOrientDiff, false, "keepOrientationDifference", "keep the initial difference in orientation (only for rigids)"))
+    , d_load(initData(&d_load, 1.0_sreal, "load", "Apply this factor to the constraint force to enable incremental loading. This value should be in the interval [0.0, 1.0]."))
     , l_topology1(initLink("topology1", "link to the first topology container"))
     , l_topology2(initLink("topology2", "link to the second topology container"))
 {
+
+    c_loadCallback.addInput(&d_load);
```

Comment:
I am not sure this is the way to handle callbacks. See for example https://github.com/sofa-framework/sofa/blob/bf56e5435e8ef93b68d0ccf0b47fa07dc0114718/Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/BaseLinearElasticityFEMForceField.inl#L39. I don't know the differences between both methods. I think @hugtalbot is a callback expert.